docs/Getting Started.md

```diff
@@ -0,0 +1,171 @@
+# Getting Started
+Welcome to Inductiva, where large-scale simulations meet simplicity, through a 
+seamless, user-friendly interface that brings cutting-edge computational resources 
+to your fingertips—effortlessly.
+
+This section is designed to provide you with all the necessary information you 
+need to get started with Inductiva API. Here, we've condensed everything so you 
```

Comment:
I like the first sentence, but I would remove the second sentence. No need for that.